R/FilterState.R

```diff
@@ -1664,10 +1664,23 @@ RangeFilterState <- R6::R6Class( # nolint
     # @return numeric(3) with names min, max, step - relevant for sliderInput
     get_pretty_range_inputs = function(values) {
       v_pretty_range <- pretty(values, n = 100)
+      min <- min(v_pretty_range)
```

Comment:
Think about a separate private method/utils function which could be properly tested and the code more clear. Even linked shiny use such `findStepSize`.